rules/import_middleman.bzl

```diff
@@ -259,6 +258,11 @@ def _file_collector_rule_impl(ctx):
         dep_cc_infos = [dep[CcInfo] for dep in ctx.attr.deps if CcInfo in dep]
         cc_info = cc_common.merge_cc_infos(direct_cc_infos = [], cc_infos = dep_cc_infos)
         additional_providers.append(cc_info)
+    else:
+        cc_info = objc_provider_utils.merge_cc_info_providers(
+            cc_info_providers = [dep[CcInfo] for dep in ctx.attr.deps],
```

Comment:
Do you also need `if CcInfo in dep` here similar to above?

Reply:
@thiagohmcruz good catch - if all the `deps` of the middleman'd rule were required to have a `CcInfo` it wouldn't be necessary - but there might have been a reason to add it originally so will add it here too.